sound/soc/intel/boards/sof_sdw.c

```diff
@@ -1161,81 +1170,44 @@ static bool is_unique_device(const struct snd_soc_acpi_link_adr *adr_link,
 	return true;
 }
 
-static int create_codec_dai_name(struct device *dev,
-				 const struct snd_soc_acpi_link_adr *adr_link,
-				 struct snd_soc_dai_link_component *codec,
-				 int offset,
-				 struct snd_soc_codec_conf *codec_conf,
-				 int codec_count,
-				 int *codec_conf_index,
-				 int adr_index,
-				 int dai_index)
+static int fill_codec_dlc(struct device *dev,
```

Comment:
Although looking at this more I do wonder if maybe this name could use being a little longer. It only really relates to the sdw stuff, will rethink for the next spin.